src/app/tests/suites/certification/Test_TC_DGGEN_2_1.yaml

```diff
@@ -57,7 +64,7 @@ tests:
               - name: "expectedValue"
                 value: "y"
 
-    - label: "Step 1b: Wait for the commissioned device to be retrieved"
+    - label: "Step 1b: Recommission the DUT after a factory reset"
```

Comment:
This does not solve the actual issue from https://github.com/project-chip/connectedhomeip/issues/29331, does it?  WaitForCommissionee does NOT re-commission the device.  You need to use PairWithCode here, no?

Was this even tested?

Reply:
@bzbarsky-apple PairWithCode will not work here as it expects the Payload value which is generated while launching the app. YAML which we don't have support to obtain the generated payload value. Issue [26127](https://github.com/project-chip/connectedhomeip/issues/26127) created for the same.

Reply:
@bzbarsky-apple  We will update the prompt logs with the comment mentioning to  
```
 Reset DUT to factory defaults
 Set the DUT to Discoverable mode
```
 What is your thoughts on this?

Reply:
@manjunath-grl This is going to need to use the factory-default setup code.  This has nothing to do with https://github.com/project-chip/connectedhomeip/issues/26127, which is about enhanced, not basic, commissioning windows.

This test should be taking the relevant setup code as a test parameter, defaulting to the setup code for the in-CI app.  TH has that code somewhere, because it could commision the device initially.

Reply:
Added Recommission step after factory-reset as discussed.